Tests/FluentAssertions.Specs/Formatting/FormatterSpecs.cs

```diff
@@ -8,13 +10,21 @@
 using FluentAssertions.Formatting;
 using FluentAssertions.Specs.Common;
 using Xunit;
+using Xunit.Abstractions;
 using Xunit.Sdk;
 
 namespace FluentAssertions.Specs.Formatting
 {
     [Collection("FormatterSpecs")]
     public class FormatterSpecs
     {
+        private readonly ITestOutputHelper outputHelper;
+
+        public FormatterSpecs(ITestOutputHelper outputHelper)
+        {
+            this.outputHelper = outputHelper;
+        }
```

Comment:
❓ What does these do?

Reply:
Removed them. That's the only way in xUnit to dump some output in the unit test results.